BrainPortal/lib/cbrain_task_generators/templates/task_params.html.erb.erb

```diff
@@ -454,7 +454,7 @@ $(function () {
   parameters.delegate('.tsk-prm-add', 'click', function () {
     var name = $(this)
       .siblings('.tsk-prm-in')
-      .data('name');
+      .attr('name');
```

Comment:
Be careful here. If the parameter is optional it may not have a `name` attribute. See [task_params.html.erb.erb:385](https://github.com/glatard/cbrain/blob/702a295bf4a4c0b248223797ab5fe6537da4b8ea/BrainPortal/lib/cbrain_task_generators/templates/task_params.html.erb.erb#L385). Might be easier to do `name += '[]'` in the `<% %>` block where `values` is processed and drop this hunk.